compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1015,7 +1015,7 @@ OMR::Power::TreeEvaluator::l2mEvaluator(TR::Node *node, TR::CodeGenerator *cg)
 TR::Register*
 OMR::Power::TreeEvaluator::v2mEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   return TR::TreeEvaluator::passThroughEvaluator(node, cg);
```

Comment:
> on Power, vectors and masks reside in the same register it can be a passThrough evaluator

That doesn't sound right to me. This opcode is is meant for converting a mask, in the form a boolean array (when 16+ lanes are present) into a mask. The mask (as a vector) cannot be in the form of a boolean array. Each lane should be all 1's or all 0's. Furthermore, the number of boolean[] elements are 8-bit. What if the elements in your mask operations are 16-bit? For instance, vmaddVector256Int16, it would have 16 elements, therefore, the vector you load is Vector128Int8 -- half the size, and incorrect lane type.